packages/relay/src/lib/eth.ts

```diff
@@ -149,6 +151,18 @@ export class EthImpl implements Eth {
    */
   private readonly chain: string;
 
+  /**
+   * The histogram used to track the number of active eth_call requests.
+   * @private
+   */
+  private ethCallHistogram: Histogram;
+
+  /**
+   * The histogram used to track the number of active eth_call requests.
```

Comment:
I think instead we should have one metric and the labels should include `method` and `function`.
Also maybe a Counter instead of a Histogram as it takes up less space and i'm not sure we're interested in the additional Histogram benefits yet

Reply:
Done.